src/table/components/virtualized-table/Table.tsx

```diff
@@ -33,26 +33,29 @@ const Table = (props: TableProps) => {
     }),
     [layout, theme.name()] // eslint-disable-line react-hooks/exhaustive-deps
   );
+  const headerRowHeight = getHeaderRowHeight(headerStyle);
+  const bodyRowHeight = getBodyRowHeight(bodyStyle);
+  const headerAndTotalsHeight = totals.atTop || totals.atBottom ? headerRowHeight + bodyRowHeight : headerRowHeight;
```

Comment:
I only find the usage of `getHeaderRowHeight` and `getHeaderRowHeight` here. I would hookify them and return these three variables, then the logic is easier to test.

Reply:
I see no value in creating a hook for it as no react hooks are being used. But what I can do is more them to a util function that returns all 3 values? It would possibly reduce the number of lines of code in this file

Reply:
that sounds good